gdist.py

```diff
@@ -0,0 +1,76 @@
+from ctypes import POINTER, CDLL, c_int, c_uint, c_double
+import glob
+
+import numpy as np
+import scipy
+
+libfile = glob.glob('build/*/gdist*.so')[0]
+
+lib = CDLL(libfile)
+
+
+class Gdist(object):
+    def __init__(self):
+        lib.compute_gdist.argtypes = [
```

Comment:
Strictly speaking, setting `argtypes` should happen once in the lifetime of the `lib` object, so it would be preferable to do this at the module level or do the `lib` creation in a function which is then called from `Gdist.__init__`, or just create the `lib` in the `Gdist.__init__` but not a mix of the three.

Reply:
Thanks!